README.md

```diff
@@ -79,306 +79,273 @@ int main()
+ctest --test-dir build
 ```
 
-</details>
+> [!IMPORTANT]
+>
+> Beman projects are
+> [passive projects](https://github.com/bemanproject/beman/blob/main/docs/BEMAN_STANDARD.md#cmake),
+> therefore,
+> you will need to specify C++ version via `CMAKE_CXX_STANDARD`
+> when manually configuring the project.
+
+### Project specific configure arguments
+
+When configuring the project manually,
+you can pass an array of project specific CMake configs to customize your build.
+
+Project specific options are prefixed with `BEMAN_EXEMPLAR`.
+You can see the list of available options with:
+
+```bash
+cmake -LH | grep "BEMAN_EXEMPLAR" -C 2
+```
 
 <details>
-<summary> Disable tests build </summary>
 
-To build this project with tests disabled (and their dependencies),
-simply use `BEMAN_EXEMPLAR_BUILD_TESTING=OFF` as documented in upstream [CMake documentation](https://cmake.org/cmake/help/latest/module/CTest.html):
+<summary> Details of CMake arguments. </summary>
 
-```shell
-cmake -B build -S . -DBEMAN_EXEMPLAR_BUILD_TESTING=OFF
+#### `BEMAN_EXEMPLAR_BUILD_TESTS`
+
+Enable building tests and test infrastructure. Default: ON.
+Values: { ON, OFF }.
+
+You can configure the project to have this option turned off via:
+
+```bash
+cmake -B build -S . -DCMAKE_CXX_STANDARD=20 -DBEMAN_EXEMPLAR_BUILD_TESTS=OFF
 ```
 
+> [!TIP]
+> Because this project requires Google Tests as part of its development
+> dependency,
+> disable building tests avoids the project from pulling Google Tests from
+> GitHub.
+
+#### `BEMAN_EXEMPLAR_BUILD_EXAMPLES`
+
+Enable building examples. Default: ON. Values: { ON, OFF }.
+
 </details>
 
 ## Integrate beman.exemplar into your project
 
-<details>
-<summary> Use beman.exemplar directly from C++ </summary>
-<!-- TODO Darius: rewrite section!-->
-
-If you want to use `beman.exemplar` from your project,
-you can include `beman/exemplar/*.hpp`  files from your C++ source files
+To use `beman.exemplar` in your C++ project,
+you should include relevant headers from `beman.exemplar` in your source files.
 
-```cpp
+```c++
 #include <beman/exemplar/identity.hpp>
 ```
 
-and directly link with `libbeman.exemplar.a`
+You will then link your project to `beman.exemplar`.
 
-```shell
-# Assume /opt/beman.exemplar staging directory.
-$ c++ -o identity_usage examples/identity_usage.cpp \
-    -I /opt/beman.exemplar/include/ \
-    -L/opt/beman.exemplar/lib/ -lbeman.exemplar
+### Produce beman.exemplar static library locally
+
+You can include exemplar's headers locally
+by producing a static `libbeman.exemplar.a` library.
+
+```bash
+cmake --workflow --preset gcc-release
+cmake --install build/gcc-release --prefix /opt/beman.exemplar
 ```
 
-</details>
+This will generate such project structure at `/opt/beman.exemplar`.
 
-<details>
-<summary> Use beman.exemplar directly from CMake </summary>
+```txt
+/opt/beman.exemplar
+├── include
+│   └── beman
+│       └── exemplar
+│           └── identity.hpp
+└── lib
+    └── libbeman.exemplar.a
+```
```

Comment:
We should just remove this section. We don't want to support arbitrary `-lbeman.exemplar` and such. This is especially true for libraries that will actually have link time dependencies for release builds.

Install instructions are good to document, but that would be for people maintaining these packages in Conan, vcpkg, Debian, or a similar project. They would go in a packaging guide, if we wanted to support one of those.

Reply:
I agree with this, I will go ahead and remove this section.

Reply:
Actually, I am not sure, `cmake --install` is a command we maintain, should I document this differently?